assets/src/application/ui/forms/espressoForm/renderers/FieldRenderer.tsx

```diff
@@ -18,8 +19,10 @@ const FieldRenderer: React.FC<FieldRendererProps> = (props) => {
 		return <MappedField {...rest} />;
 	}
 
+	const tooltipKey = info ? uuidv4() : null;
+
 	const fieldInfo = info ? (
-		<Tooltip placement='right' title={info}>
+		<Tooltip placement='right' title={info} id={tooltipKey}>
```

Comment:
TypeScript doesn't like this because AntD's typing for this... **does not allow an ID attribute !?!?!?**